server/neptune/workflows/internal/terraform/workflow.go

```diff
@@ -212,7 +231,7 @@ func (r *Runner) Run(ctx workflow.Context) error {
 	var response *activities.GetWorkerInfoResponse
 	err := workflow.ExecuteActivity(ctx, r.TerraformActivities.GetWorkerInfo).Get(ctx, &response)
 	if err != nil {
-		return errors.Wrap(err, "getting worker info")
+		return r.toExternalError(err, "getting worker info")
```

Comment:
Why are we doing this here? What if the caller doesn't know about this and decides to wrap the error? This will then break.  Conversion needs to happen right at the bounds of the workflow.

Reply:
oops this is my bad. carry on.